src/funding/paylater/config.jsx

```diff
@@ -49,6 +56,19 @@ function getLabelText(fundingEligibility : FundingEligibilityType) : ?string {
         labelText = '4X PayPal';
     }
 
+    if(getLocale().includes("US")) {
+        const payload = {
+            state_name: "elmo_check",
+            transition_name: "process_elmo_check",
+            experimentation_experience: SECOND_BTN_EXP.EXPERIMENTATION_EXPERIENCE,
+            experimentation_treatment: paylater?.products?.payIn4?.reasons.includes(SECOND_BTN_EXP.REASON) ? SECOND_BTN_EXP.TREATMENT_ID : SECOND_BTN_EXP.CONTROL_ID,
```

Comment:
This check can't be used, `reasons` isn't part of the query and also can't be accessed externally